inspect/config.js

```diff
@@ -0,0 +1,28 @@
+// Handler for onOptions event, invoked after we send ready message
+grist.onOptions(options => {
+  if (options?.name) {
+    document.getElementById("textbox").value = options.name;
+  }
+});
+
+// Save button handler
+async function saveConfig() {
+  const name = document.getElementById('textbox').value;
```

Comment:
For this to serve as a good example, maybe we can factor out the reusable parts of a config right in this first PR on it. It seems to me that as long as we can designate some standard location for the config-specific CSS and JS files, we could move out the CSS from HTML to there, and move this JS file and make it generic.

To make it generic, we can save the entire form, e.g. using a function like `formDataToObj` from Grist's https://github.com/gristlabs/grist-core/blob/8853e095bb09afdfaa984de271736b6edd36114c/app/client/lib/uploads.ts#L191.

We could use this JS file for most setup, including adding the CSS to the page, and setting event handlers on everything, so that the config HTML has just `<script src="./config.js"></script>` and a plain-html form with input names that correspond to the keys of the data object, and some standard element IDs (for the form and for the buttons, if that's even needed).